backend/pluto/pyproject.toml

```diff
@@ -8,6 +8,8 @@ readme = "README.md"
 [tool.poetry.dependencies]
 python = "^3.10"
 psycopg2 = "^2.9.6"
+flask = "^2.3.1"
+types-requests = "^2.30.0.0"
```

Comment:
These can be moved to the `dev` dependency group, since they are not used during runtime. Manually remove these lines, then do `poetry add 'flask^2.3.1' --group dev` or something like that.